api/src/main/java/org/openmrs/module/fhir/api/condition/ObsConditionStrategy.java

```diff
@@ -13,15 +13,39 @@
  */
 package org.openmrs.module.fhir.api.condition;
 
+import ca.uhn.fhir.model.dstu2.composite.CodeableConceptDt;
+import ca.uhn.fhir.model.dstu2.composite.CodingDt;
+import ca.uhn.fhir.model.dstu2.composite.ResourceReferenceDt;
 import ca.uhn.fhir.model.dstu2.resource.Condition;
+import ca.uhn.fhir.model.primitive.DateDt;
+import ca.uhn.fhir.model.primitive.IdDt;
+import org.apache.commons.lang.StringUtils;
+import org.openmrs.ConceptMap;
+import org.openmrs.Obs;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.fhir.api.util.FHIRConstants;
+import org.openmrs.module.fhir.api.util.FHIRUtils;
 
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
 import java.util.List;
 
 public class ObsConditionStrategy implements GenericConditionStrategy {
 
 	@Override
 	public Condition getConditionById(String uuid) {
-		return null;
+		String conceptsAsConditions = Context.getAdministrationService().getGlobalProperty("fhir.concepts.conditions");
+		List<String> concepts = new ArrayList<String>();
```

Comment:
cant we have integer array or something without converting this to string always